addons/certificate/tools/certificate_adapter.py

```diff
@@ -0,0 +1,49 @@
+from base64 import b64decode
+import requests
+from OpenSSL.crypto import load_certificate, load_privatekey, FILETYPE_PEM
+from urllib3.contrib.pyopenssl import inject_into_urllib3
+from urllib3.util.ssl_ import create_urllib3_context
```

Comment:
Not a big deal at all but I like being as accurate as possible with imports so If you need to push again, could I ask you to take a minute to fix your imports ?

```suggestion
import requests
from OpenSSL.crypto import FILETYPE_PEM, load_certificate, load_privatekey
from base64 import b64decode
from urllib3.contrib.pyopenssl import inject_into_urllib3
from urllib3.util.ssl_ import create_urllib3_context
```

Reply:
Sorry, my mind is traumatized from pre-`ruff` times, when it was a major sin to introduce "cosmetic" changes in stable, and "unjustified" diff-s.